Report/Core/MeasureReportSubmissionBundler.cs

```diff
@@ -200,6 +200,7 @@ private void BundleSharedPatientLineLevel(Bundle bundle, MeasureReport measureRe
                 measureReport.Contained.RemoveAt(0);
 
                 // remove the ext that reference the contained resource
+                measureReport.Extension.RemoveAll(e => e.Value is Extension);
```

Comment:
We should not remove all from Extension, only the ones that reference the contained resources.